client/state/login/actions.js

```diff
@@ -38,6 +38,7 @@ export const loginUser = ( usernameOrEmail, password ) => dispatch => {
 	} );
 
 	return request.post( config( 'login_url_xhr' ) )
+		.withCredentials()
```

Comment:
Here is the [documentation page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#Requests_with_credentials) - for more information.